src/components/hoc/status.js

```diff
@@ -0,0 +1,34 @@
+import { update } from 'ramda';
+import { compose, withState, withHandlers } from 'recompose';
```

Comment:
modules could be imported individually 
`import compose from 'recompose/compose'` or use babel-recompose-plugin